oiosaml/src/test/java/dk/gov/oio/saml/util/StringUtilTest.java

```diff
@@ -43,7 +55,7 @@ void testElementToString() throws Exception {
     @DisplayName("Test that input string is JSON escaped after running jsonEscape")
     @Test
     void testJsonEscape() {
-        Assertions.assertEquals("\\t\\b\\r\\n\\f\\\"\\\\TEST\\u2028\\t\u9515", StringUtil.jsonEscape("\t\b\r\n\f\"\\TEST\u2028\u0009\u9515"));
+        Assertions.assertEquals("\\\\t\\t\\b\\r\\n\\f\\\"\\\\TEST\\u2028\\t\u9515", StringUtil.jsonEscape("\\t\t\b\r\n\f\"\\TEST\u2028\u0009\u9515"));
```

Comment:
Was bad before and still is

Reply:
Feel free to elaborate and propose a solution :)

Reply:
We should create a Jira for this if there is a better solution - the best would be to depend on someone else to make this right.